Makefile

```diff
@@ -187,19 +187,42 @@ set-manifest-image:
 ## --------------------------------------
 
 RELEASE_TAG := $(shell git describe --abbrev=0 2>/dev/null)
+RELEASE_DIR := out
+
+$(RELEASE_DIR):
```

Comment:
Does this need to be phony?

Reply:
How so? It'd only create it if it doesn't exist, unless I'm missing something

Reply:
Nevermind